### PR TITLE
Add Business Logic Abuse to ASM docs

### PR DESCRIPTION
**Pull request by maxhedouin**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by maxhedouin** [comment]
Adding the do not merge until the feature is released (should be next week). Reviews are welcomed!

File: content/en/security/application_security/how-appsec-works.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ further_reading:
 
 ## Overview
 
-Datadog Application Security Management (ASM) provides observability into application-level attacks that aim to exploit code-level vulnerabilities, and into any bad actors targeting your systems.
+Datadog Application Security Management (ASM) provides observability into application-level attacks that aim to exploit code-level vulnerabilities, abuse the business logic of your application, and into any bad actors targeting your systems.
```
Comment (kayayarai) [inline content/en/security/application_security/how-appsec-works.md:21]:
```suggestion
Datadog Application Security Management (ASM) provides observability into application-level attacks that aim to exploit code-level vulnerabilities or abuse the business logic of your application, and into any bad actors targeting your systems.
```

File: content/en/security/application_security/threats/custom_rules.md [hunk 2]
```diff
@@ -24,49 +24,56 @@ further_reading:
...
+However, there are situations where you may want to customize a rule based on your environment or workload. For example, you may want to customize a detection rule that detects users performing sensitive actions from a geolocation where your business doesn’t operate.
 
 Another example is customizing a rule to exclude an internal security scanner. ASM detects its activity as expected. However, you may not want to be notified of its regularly occurring scan.
 
 In these situations, a custom detection rule can be created to exclude such events. This guide shows you how to create a custom detection rule for ASM.
 
+## Business logic abuse detection rule
+ASM offers out of the box rules to detect business logic abuse (Reset password brute forcing for example). Those rules require [adding business logic information to traces][7].
```
Comment (kayayarai) [inline content/en/security/application_security/threats/custom_rules.md:36]:
```suggestion
ASM offers out of the box rules to detect business logic abuse (for example, resetting a password through brute force). Those rules require [adding business logic information to traces][7].
```

File: content/en/security/application_security/threats/custom_rules.md [hunk 3]
```diff
@@ -24,49 +24,56 @@ further_reading:
...
+
+You can filter the rules, and identify which business logic to start tracking. Additionally, these rules can be used as a blueprint to create your own rules based on your own business logic.
```
Comment (kayayarai) [inline content/en/security/application_security/threats/custom_rules.md:38]:
```suggestion
You can filter the rules, and identify which business logic to start tracking. Additionally, you can use these rules as a blueprint to create custom rules based on your own business logic. 
```

File: content/en/security/application_security/threats/custom_rules.md [hunk 4]
```diff
@@ -24,49 +24,56 @@ further_reading:
...
+Construct an ASM query using the [same query syntax as in the ASM Trace Explorer][3]. For example, create a query to monitor login successes from outside of the United States: `@appsec.security_activity:business_logic.users.login.success -@actor.ip_details.country.iso_code:US`.
 
-Optionally, define a unique count and signal grouping. Count the number of unique values observed for an attribute in a given timeframe. The defined group-by generates a signal for each group-by value. Typically, the group-by is an entity (like user or IP). The group-by is also used to [join the queries together](#joining-queries).
+Optionally, define a unique count and signal grouping. Count the number of unique values observed for an attribute in a given timeframe. The defined group-by generates a signal for each group-by value. Typically, the group-by is an entity (like user, IP, and/or service). The group-by is also used to [join the queries together](#joining-queries).
```
Comment (kayayarai) [inline content/en/security/application_security/threats/custom_rules.md:50]:
```suggestion
Optionally, define a unique count and signal grouping. Count the number of unique values observed for an attribute in a given timeframe. The defined group-by generates a signal for each group-by value. Typically, the group-by is an entity (like user, IP, or service). The group-by is also used to [join the queries together](#joining-queries).
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Some edits for grammar and style. Please make sure I didn't mangle any meaning.

**Review by kayayarai — APPROVED** [review]

**Pull request merged**